cluster/userdata-worker.yaml

```diff
@@ -117,12 +81,44 @@ coreos:
         --tls-private-key-file=/etc/kubernetes/ssl/worker-key.pem \
         --cloud-provider=aws \
         --low-diskspace-threshold-mb=1000 \
-        --feature-gates=AllAlpha=true
+        --feature-gates=AllAlpha=true \
+        --system-reserved=cpu=100m,memory=164Mi \
+        --kube-reserved=cpu=100m,memory=282Mi
         Restart=always
         RestartSec=10
         [Install]
         WantedBy=multi-user.target
 
+    - name: kube-node-drainer.service
+      enable: true
+      command: start
+      runtime: true
+      content: |
+        [Unit]
+        Description=drain this k8s node to make running pods time to gracefully shut down before stopping kubelet
+        Requires=docker.service
+        After=docker.service
+
+        [Service]
+        Type=oneshot
+        RemainAfterExit=true
+        ExecStart=/bin/true
+        TimeoutStopSec=40s
```

Comment:
I guess this will limit the `terminationGracePeriodSeconds` to <40. We should document this limit and evaluate if this is a reasonable limit for our users.